packages/toolpad-app/src/appDomMigrations/index.ts

```diff
@@ -0,0 +1,10 @@
+import invariant from 'invariant';
+import v1 from './v1';
+
+const versions = new Map([[1, v1]]);
+
+export const latestVersion = Array.from(versions.keys()).pop();
+
+invariant(latestVersion, 'No migrations found');
```

Comment:
```suggestion
```

Reply:
This helps with the type error on the `versions.get(latestVersion)` in the last line, but I agree with this